src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -916,7 +916,7 @@ public void onGestureStatus(Status status) {
     unsubscribe(python.getName(), "publishStatus", this.getName(), "onGestureStatus");
   }
 
-  public void onIKServoEvent(ServoData data) {
+  public void onIKServoEvent(ServoEvent data) {
     if (vinMoovApp != null) {
       vinMoovApp.updatePosition(data);

```

Comment:
This shouldn't be servoEvent.. this should be the JoinAngle IK / Encoder data with position.